Pipfile

```diff
@@ -25,8 +30,11 @@ greenlet = "*"
 dateutils = "*"
 requestsdefaulter = "*"
 flask-zipkin = "*"
+jwcrypto = "*"
```

Comment:
You shouldn't need to add jwcrypto as a top level dependency if it wasn't before

Reply:
Yes I agree but... running `pipenv lock` without it results in 
``` 
"jwcrypto": {
    "hashes": [],
    "version": "==0.5"
```
And I want to avoid pinning the top level dependency to a version that produces the correct hashes.

Reply:
Found the problem: https://github.com/ONSdigital/sdc-cryptography/blob/450f022983ae4912d002964ee2abb41b81fa8f2e/requirements.txt#L2

Luckily it's a repo we manage...